pkg/event/logging_test.go

```diff
@@ -30,7 +30,7 @@ func TestMuxComponentLoggingMW(t *testing.T) {
 
 	// Event.
 	mockMuxComponent.EXPECT().Event(ctx, "Event", event).Return(nil).Times(1)
-	mockLogger.EXPECT().Log("unit", "Event", "type", "Event", "correlation_id", corrID, "took", gomock.Any()).Return(nil).Times(1)
+	mockLogger.EXPECT().Info("unit", "Event", "type", "Event", "correlation_id", corrID, "took", gomock.Any()).Return(nil).Times(1)
```

Comment:
Do we really want to mock loggers ? (We could use NoopLogger rather than a mocked logger)

Reply:
As we are testing the logging, sounds good to me to use a mock to ensure we really call the logger, but I agree for the others cases